.github/workflows/code-review.yml

```diff
@@ -19,12 +19,20 @@ jobs:
     steps:
       - name: 👀 Checkout
         uses: actions/checkout@v4
+      - name: ⬢ Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
       - name: ➕ Add `bin` to GITHUB_PATH
         run: echo "$(pwd)/bin" >> $GITHUB_PATH
       - name: ♻️ Restore caches
         uses: ./.github/actions/expo-caches
         with:
           yarn-tools: 'true'
+      - name: 🧶 Yarn install
+        working-directory: tools
+        if: steps.expo-caches.outputs.yarn-tools-hit != 'true'
+        run: yarn install --immutable
```

Comment:
quick question: is that necessary? since the workflow runs on people's branch. wondering if install node_modules would introduce security risk

Reply:
@Kudo I think that this PR is always running from `main` workflow file (`pull_request_target` trigger vs `pull_request`), so there should not be the security risk involved in there. 

The problem which we have was that `et` automatic "Yarning" step was not triggered properly, even with manual cache invalidation. I guess if we remove this step now, it will retain working until there is an another cache related issue on the CI.

LMK if you have in mind an another approach to fixing that problem! Happy to improve the solution if possible.

Reply:
> I think that this PR is always running from main workflow file (pull_request_target trigger vs pull_request), so there should not be the security risk involved in there.

yes you are right. tested with https://github.com/expo/expo/pull/35462 and it doesn't checkout pr's head. then it should be safe. thanks for fixing the ci problem as well as sharing me the correct info for `pull_request_target`. you are the best as always 👏